### DISCUSSION TITLE
Multiarch Support?

### DISCUSSION THREAD

**Discussion by lcnittl** [timmo001/system-bridge#953] [General]
Are there any plans to pick up multiarch support again? Like mentioned here:
https://github.com/timmo001/system-bridge/issues/101
https://github.com/timmo001/system-bridge/pull/105

arm64 deb files would be highly appreciated for raspberry pi installation.

Cheers, lcnittl

**Comment by timmo001** [comment]
The app has the potential to be built for arm(64) using pkg (`yarn package` in an arm device).

QEMU would have to be used, and the speed of the build may be a concern so this may need to be in a separate workflow
